srp.go

```diff
@@ -134,32 +171,35 @@ type SRP struct {
 var bigZero = big.NewInt(0)
 var bigOne = big.NewInt(1)
 
-/*
-NewSRP creates an SRP object and sets up defaults.
+/* NewSRPClient sets up an SRP object for a client.
```

Comment:
Yeah. NewSRPServer and NewSRPClient are much friendlier. Although _I_ liked the abstraction for my own reasons, it probably isn't helpful for anyone who doesn't deeply understand the differences between the client and server end.